src/scripts/browserdeviceprofile.js

```diff
@@ -18,12 +18,27 @@ define(['browser'], function (browser) {
             }
         }
 
-        // Unfortunately haven't yet found a canPlayType for proper detection
-        if (browser.iOS && (browser.iOSVersion || 0) >= 11) {
-            return true;
+        if (browser.ps4) {
+            return false;
+        }
+
+        if (protocol === 'hls') {
+
+            //safari seems to be lying about this
+            if (browser.iOS || browser.safari) {
+                return false;
+            }
+
+            return !!videoTestElement.canPlayType &&
+            (videoTestElement.canPlayType('video/mp2t; codecs="hvc1.1.L0.0"').replace(/no/, '') ||
+            videoTestElement.canPlayType('video/mp2t; codecs="hev1.1.L0.0"').replace(/no/, '') ||
+            videoTestElement.canPlayType('video/mp2t; codecs="hev1.1.2.L150"').replace(/no/, ''));
         }
 
-        return !!(videoTestElement.canPlayType && videoTestElement.canPlayType('video/hevc; codecs="hevc, aac"').replace(/no/, ''));
+        return !!videoTestElement.canPlayType &&
+        (videoTestElement.canPlayType('video/mp4; codecs="hvc1.1.L0.0"').replace(/no/, '') ||
+        videoTestElement.canPlayType('video/mp4; codecs="hev1.1.L0.0"').replace(/no/, '') ||
+        videoTestElement.canPlayType('video/mp4; codecs="hev1.1.2.L150"').replace(/no/, ''));
```

Comment:
1) this seems to be a direct copy-paste of lines above - there's no need for that
2) where did you get these levels? can you explain their magic here?

Reply:
`video/mp4; codecs="hev1.1.0.L120"` for new Edge Chromium and iOS.
`canPlayType('video/hevc; codecs="hevc,aac"')` output NO on new Edge and iOS.
`hvc1` and `hev1` are two different names of writing video tags.

refer:
https://cconcolato.github.io/media-mime-support/
https://chromium.googlesource.com/chromium/src.git/+/master/media/base/video_codecs_unittest.cc#450